trade-finance-manager-ui/templates/index.njk

```diff
@@ -56,5 +56,6 @@
     <script src="/assets/js/mojFrontend.js" type="text/javascript" integrity="sha512-VJYfsojYZY4stpSFP22EQ/CXqqrW9BWYWKaiEnYGmyw22rggV4DCe+Lzn8L7Gt+bRq3AyWuNYFadfrCmao0o+A==" crossorigin="anonymous"></script>
     <script src="/assets/js/disableFormSubmitOnSubmission.js" type="text/javascript" integrity="sha512-tCINlYTcYERel2bdysTMl5KaemDdEyog5JSS3hYQqHS6ZIyT4ATLu4oTPerbZ+vAOn4AAzDEsRBSQVacGhoieQ==" crossorigin="anonymous"></script>
     <script src="/assets/js/enableSelectAllTableCheckbox.js" type="text/javascript" integrity="sha512-ER/5raNAJX5VTcyrRBjVExC/f39WaVY4r8k1CyGqJUad95QZ0A1H6zHtsFpdf0JDBLV0acXzSA1B2mkKC8g5Tg==" crossorigin="anonymous"></script>
+  <script src="/assets/js/enableFindUtilisationReportsByYearDropdownAutocomplete.js" integrity="sha512-b1kS23cn0PI6KoBLTw3xeNJ2qi11c5nGrnDP3zk9DCuayz0/fu4PbzbUiqPaD5062eI9KJRp1e1ZvApQLGjpug==" type="text/javascript" crossorigin="anonymous"></script>
```

Comment:
`findYearlyUtilisationReportsDropdown` might be a suitable file name.

Reply:
I think that name is confusing because it suggests that the script is "finding the yearly utilisation reports dropdown", which doesn't make sense. It also removes the intentional linking to the `find utilisation reports by year` page by including `FindUtilisationReportsByYear` in the name

Reply:
In that case can be just `findUtilisationReportsByYear` please and what the script actually does can be explained in-depth as comments please.

Reply:
Again, that name implies that the script performs an action ("_finds_ utilisation reports by year") which can lead to confusion even with a comment in the file as the comment will only be in the source file; not the actually built script (ie. `find-utilisation-reports-by-year.ts` would have the comment, but `public/js/findUtilisationReportsByYear.js` would not)

Reply:
* The source file still needs to have comments.
* Having reviewed the file it seems to be adding bank id to year input unless I have misunderstood it can it be rather called either `addBankIdToYearInput` or something along those lines?

Reply:
It does not add a bank id to the year input. This may be worth taking offline as it seems the missing context around this ticket is causing confusion

Reply:
Renamed to `enableFindReportsByYearDropdown`.